src/Tickets/Seating/Service/Layouts.php

```diff
@@ -125,7 +128,45 @@ public function get_in_option_format() {
 
 		return $layouts;
 	}
-
+	
+	/**
+	 * Fetches all the Layouts from the database.
+	 *
+	 * @since TBD
+	 *
+	 * @return Layout_Card[] Array of layout card objects.
+	 */
+	public function get_in_card_format() {
+		if ( ! $this->update() ) {
+			return [];
+		}
+		
+		$cache_key    = 'option_layout_card_objects';
+		$layout_cards = wp_cache_get( $cache_key, 'tec-tickets-seating' );
+		
+		if ( ! ( $layout_cards && is_array( $layout_cards ) ) ) {
+			$layout_cards = [];
+			foreach ( Layouts_Table::fetch_all() as $row ) {
+				$layout_cards[] = new Layout_Card(
+					$row->id,
+					$row->name,
+					$row->map,
+					$row->seats,
+					$row->screenshot_url
+				);
+			}
+			
+			wp_cache_set(
```

Comment:
Let's avoid this caching.

You're just moving data from one database, to another.
The query is simple enough and fast enough to have equivalent times, the data handling is minor and not involving complicate computations that you would save with a caching.

Instead of caching this value, memoize it.